packages/compass-aggregations/src/components/focus-mode/focus-mode.tsx

```diff
@@ -80,6 +90,97 @@ type FocusModeProps = {
   onCloseModal: () => void;
 };
 
+const INPUT_RESIZER_PROPS = {
```

Comment:
Feel free to keep it as is, but I am wondering why are we keeping this out of the actual render? Jumping back and forth between render method and prop definitions make it a bit awkward to read

Reply:
to avoid react. but i can use `useMemo` and have it there.

Reply:
Is this a recommended way to use this library or causing performance issues otherwise? I'm assuming that some of those objects being "re-created" every render would be handled by the component itself correctly and will not cause issues and a lot of these are primitive. I would suggest not to over-optimize like that unless you have a more or less real case for this